### PR TITLE
Stop generating InteractiveComponents VSIX

### PR DESCRIPTION
**Pull request by tmat**
Fixes https://github.com/dotnet/roslyn/issues/34002

### CODE REVIEW

**Comment by tmat** [comment]
@CyrusNajmabadi FYI

File: src/VisualStudio/VisualStudioInteractiveComponents/Roslyn.VisualStudio.InteractiveComponents.csproj [hunk 1]
```diff
@@ -73,11 +65,8 @@
...
   </ItemGroup>
-  <ItemGroup>
-    <None Include="source.extension.vsixmanifest">
-      <SubType>Designer</SubType>
-    </None>
-  </ItemGroup>
+
+  <Import Project="$(VSToolsPath)\vssdk\Microsoft.VsSDK.targets"/>
 
   <Target Name="VsixSourceItemsOutputGroup"
```
Comment (jasonmalinowski) [inline src/VisualStudio/VisualStudioInteractiveComponents/Roslyn.VisualStudio.InteractiveComponents.csproj:71]: Can you delete this bit too?

**Comment by CyrusNajmabadi** [comment]
Would love to get this in asap.  It makes f5 debugging such a PITA.

File: src/VisualStudio/VisualStudioInteractiveComponents/Roslyn.VisualStudio.InteractiveComponents.csproj (hunk 1 above)
Comment (tmat) [reply src/VisualStudio/VisualStudioInteractiveComponents/Roslyn.VisualStudio.InteractiveComponents.csproj:71]: No, this is important. It's called from the referencing project.

**Comment by tmat** [comment]
@CyrusNajmabadi 
> Would love to get this in asap. 

Your wish is granted. :)

**Comment by kzu** [comment]
This broke the RoslynDeployment.vsix since now there's a dependency that's missing... I'm guessing testing feature branches using that isn't very popular ;). 

Fixed in #45159 .

**Pull request merged**